components/browser/engine-gecko-beta/README.md

```diff
@@ -12,6 +14,46 @@ Use gradle to download the library from JCenter:
 implementation "org.mozilla.components:engine-gecko-beta:{latest-version}
```

Comment:
I'm not quite sure 🤔, if we should also include as a step adding the dependency of 
**geckoview**, I couldn't resolve **GeckoRuntime** reference by just following the instructions
I had to included. 
> implementation "org.mozilla.geckoview:{latest-version}"

Reply:
Yeah, that's a great idea. And we should link to the CHANGELOG that mentions compatible versions. I'll file a follow-up.

Reply:
👍